{
  "_annotation": {
    "cve_id": "CVE-2024-45606",
    "generated_from": "https://raw.githubusercontent.com/anchore/cve-data-enrichment/main/data/anchore/2024/CVE-2024-45606.json",
    "reason": "Added CPE configurations because not yet analyzed by NVD."
  },
  "cve": {
    "configurations": [
      {
        "nodes": [
          {
            "cpeMatch": [
              {
                "criteria": "cpe:2.3:a:sentry:sentry:*:*:*:*:*:python:*:*",
                "matchCriteriaId": "FE0DE760-4168-5BDD-A2BF-6D95A685D5C1",
                "versionEndExcluding": "24.9.0",
                "versionStartIncluding": "23.4.0",
                "vulnerable": true
              }
            ],
            "negate": false,
            "operator": "OR"
          }
        ]
      }
    ]
  }
}